evalml/utils/woodwork_utils.py

```diff
@@ -144,6 +144,13 @@ def downcast_nullable_types(X, ignore_null_cols=True):
     new_ltypes = {
         col: "Boolean" for col in X_bool_nullable_cols if col in non_null_columns
     }
+    new_ltypes.update(
+        {
+            col: "BooleanNullable"
+            for col in X_bool_nullable_cols
+            if col not in new_ltypes
+        },
+    )
```

Comment:
great idea! My thought here was that it would save time if it didn't have to set the logical type for every column but if we use 'init' I don't think we would be saving time. I'm discussing with @thehomebrewnerd and @ParthivNaresh about using 'set_types' here and it could be different with that.

Reply:
OK, that would be fine. But yeah I think not setting the type with `init` will actually take more time as woodwork would then try to infer it.